src/__swaps__/screens/Swap/components/SearchInput.tsx

```diff
@@ -131,10 +127,9 @@ export const SearchInput = ({
 
   const onPaste = () => {
     Clipboard.getString().then(text => {
-      if (text.length < 10 || isAddress(text)) {
-        pastedSearchInputValue.value = text;
-        useSwapsStore.setState({ outputSearchQuery: text });
-      }
+      const v = text.trim().slice(0, 42);
```

Comment:
can you leave an inline comment why this is necessary? I know that contract addresses are 42-bits long but others reading this might not know why this is here.